### PR TITLE
Add `jupyter-releaser`

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Fixes https://github.com/jupyter-server/jupyter_releaser/issues/142

### CODE REVIEW

File: recipes/jupyter-releaser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+about:
+  home: https://jupyter.org
+  summary: Jupyter Releaser for Python and/or npm packages.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - blink1073
```
Comment (jtpio) [inline recipes/jupyter-releaser/meta.yaml:62]: cc @blink1073 if you are fine being listed here

**Comment by jtpio** [comment]
Ah looks like a couple of dependencies are missing:

image

**Comment by blink1073** [comment]
👍🏼 thanks!

**Comment by jtpio** [comment]
To package on conda:

- [x] `tbump`: https://anaconda.org/conda-forge/tbump
- [ ] `github-activity`: https://github.com/conda-forge/staged-recipes/pull/17741
- [ ] `pypiserver`
- [ ] `ghapi`

**Comment by jtpio** [comment]
Looks like CI is all green now!

**Comment by jtpio** [comment]
cc @conda-forge/help-python this is ready to go, thanks!

**Pull request merged**